### PR TITLE
Simple demo creation

### PR DESCRIPTION
**Pull request by colbycheeze**
Completes #49, and #55, and invalidates #47.

Users can now easily create a demo w/o entering any information, or paste a guid to join an existing session.

screen shot 2016-09-27 at 2 29 02 pm

Once a demo has been visited, the GUID is stored in localStorage and users are automatically redirected to the dashboard instead of hitting the create-demo route.

screen recording 2016-09-27 at 03 18 pm

### CODE REVIEW

**Comment by rvennam** [comment]
I like it. Much simpler.

File: src/services/index.js [hunk 1]
```diff
@@ -18,10 +18,10 @@ export const callApi = (endpoint, {
     return json;
   });
 
-export const createDemo = (name, email) =>
+export const createDemo = () =>
   callApi('demos', {
     method: 'POST',
-    body: { name, email },
+    body: { name: 'demo' },
```
Comment (l2fprod) [inline src/services/index.js]: do you use the name for anything?

I guess you kept it cause it is required in the Controller API.

if you don't use it we can remove it from the API too.

**Comment by l2fprod** [comment]
@colbycheeze how do I log out of the demo and create a new one? Let's say I want to restart the scenario with a fresh env.

File: src/services/index.js (hunk 1 above)
Comment (colbycheeze) [inline src/services/index.js]: Correct, the api requires it. You can remove it or make it optional in the api as it shouldn't be needed for our use case.

**Comment by colbycheeze** [comment]
> @colbycheeze how do I log out of the demo and create a new one? Let's say I want to restart the scenario with a fresh env.

I didn't code that in, but if we think it's important there can be two ways of us allowing that.
 1. Instead of redirecting on create-demo screen...can just auto-fill the existing guid (thus giving them ability to create new demo
 2. Add an option somewhere in the header to reset/create new demo along with other "meta options" such as switching roles, creating weather events, etc (things we will eventually build to interact with the demo)

**Pull request merged**